crates/cheatcodes/src/inspector.rs

```diff
@@ -545,6 +546,7 @@ impl<DB: DatabaseExt> Inspector<DB> for Cheatcodes {
                         reverted: false,
                         deployedCode: vec![],
                         storageAccesses: vec![],
+                        depth: U256::from(0), // updated on cheatcodes::evm::get_state_diff
```

Comment:
Since depth is a `crates::Vm::AccountAccess` field, we can set it here directly as it's done on L554 (and elsewhere). So we can then get rid of the `cheatcodes::AccountAccess` struct.